homeworks/asztaloslajos/3_conditionals_loop/rock_paper_scissors.py

```diff
@@ -0,0 +1,74 @@
+"""
+rock_paper_scissors.py --- Asztalos Lajos --- 2024.10.13
+"""
+#parameters
+RPS = list(["kő","papír", "olló"])
+a_score = 0
+b_score = 0
+
+#input rounds
+i=0
+while i == 0:
+    rounds_num=int(input("Add meg a fordulók számát:"))
+    i=rounds_num % 2
+    if i==0: print("Csak páratlan számot adhatsz meg!")
+print(f"Kezdhetjük a játékot {rounds_num} fordulóval.")
+
+
+#play
+round = 0
+while round < rounds_num:
+    round += 1
+    print(f"Forduló:{round}, A:{a_score}, B:{b_score}")
+    #input first bet
+    i=False
```

Comment:
ezt a két while loopot ahol a játékosok tippjeit kéred be, lehet egy kicsit egyszerűsíteni:

```
    while True:
        a_player_bet = input(f"Az első játékos {round}. tétje:")
        if a_player_bet in RPS:
            break
        print("A megadott érték csak kő, papír vagy olló lehet!")
    #input second bet
    while True:
        b_player_bet = input(f"A második játékos {round}. tétje:")
        if b_player_bet in RPS:
            break
        print("A megadott érték csak kő, papír vagy olló lehet!")
```

Ami itt történik az az, hogy ha a megadott válasz benne van az RPS-ben akkor kilépünk a ciklusból, ha nem, akkor kiprinteljük hogy rosszat adott meg, és újra próbálkozunk a while True miatt. Persze ezt is tovább lehetne egyszerűsíteni ha function-öket használnánk, de ez a házi még nem tartozott a function-ös témakörbe.